pkg/kubelet/kubelet.go

```diff
@@ -955,8 +958,11 @@ func (kl *Kubelet) syncNodeStatus() {
 	}
 }
 
-func makeMounts(container *api.Container, podVolumes kubecontainer.VolumeMap) (mounts []kubecontainer.Mount) {
+func (kl *Kubelet) makeMounts(pod *api.Pod, container *api.Container, podVolumes kubecontainer.VolumeMap) ([]kubecontainer.Mount, error) {
+	mountEtcHostsFile := !pod.Spec.SecurityContext.HostNetwork
```

Comment:
It mounts the the host's /etc/hosts

On Wed, Oct 21, 2015 at 1:42 PM, Brian Grant <notifications@github.com>
wrote:

> In pkg/kubelet/kubelet.go
> <https://github.com/kubernetes/kubernetes/pull/16052#discussion_r42679420>
> :
>
> > @@ -955,8 +958,11 @@ func (kl *Kubelet) syncNodeStatus() {
> >  	}
> >  }
> >
> > -func makeMounts(container *api.Container, podVolumes kubecontainer.VolumeMap) (mounts []kubecontainer.Mount) {
> > +func (kl *Kubelet) makeMounts(pod *api.Pod, container *api.Container, podVolumes kubecontainer.VolumeMap) ([]kubecontainer.Mount, error) {
> > +	mountEtcHostsFile := !pod.Spec.SecurityContext.HostNetwork
>
> What does Docker do with /etc/hosts in the case of net=host?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/16052/files#r42679420>.
>